### PR TITLE
Implemented C++ Hello, World!

### PR DESCRIPTION
**Pull request by MidnightKaze**
Went ahead and added the C++ version of "Hello, World!", which is pretty similar to the C one. 

Due to some VSCode configurations the actual C++ file is generating some errors, but as far as I know it should work properly. If someone wants to confirm or give their two senses on that feel free. The part that currently pops up as an error is:

`#include <iostream>`

Again, it might just be a VSCode issue. Let me know if there's any issues!

### CODE REVIEW

File: C++/hello.cpp
[the remark this answers is not in the document]
Comment (MidnightKaze) [reply C++/hello.cpp:1]: I went ahead an changed the directory to cpp and I also added that new space to the end of that line of code :D

**Comment by MidnightKaze** [comment]
Since I resolved all of those conversations I'll leave another comment here too. I went ahead and changed C++ >> cpp as asked. I also added a new line character to the end of that "Hello, World!" so now the code is as such:

`std::cout << "Hello, World!" << std::endl;`

Let me know if anything else should be changed :D

**Review by zachflower — CHANGES_REQUESTED** [review]
@MidnightKaze one final ask! Create a `.gitignore` file in the `cpp` directory, and add `hello` to it. This will keep it so the `hello` binary can't accidentally get committed to the repository

**Comment by MidnightKaze** [comment]
I gotcha. Should be in there now!

File: cpp/hello.cpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main() {
+    std::cout << "Hello, World!" << std::endl;
```
Comment (zachflower) [inline cpp/hello.cpp:4]: Ahh! Missed this in my review (I added tests and it caught it). Should be lowercase, no punctuation:

```suggestion
    std::cout << "hello, world" << std::endl;
```

**Review by zachflower — CHANGES_REQUESTED** [review]
Sorry, final _final_ request

File: cpp/hello.cpp (hunk 1 above)
Comment (MidnightKaze) [reply cpp/hello.cpp]: It has been done. "Hello, World!" >> "hello, world"

**Review by zachflower — APPROVED** [review]

**Pull request merged**